HealthMonitoring.Persistence/SqlEndpointConfigurationStore.cs

```diff
@@ -22,8 +22,29 @@ public void SaveEndpoint(Endpoint endpoint)
             using (var conn = _db.OpenConnection())
             using (var tx = conn.BeginTransaction())
             {
-                if (conn.Execute("update EndpointConfig set MonitorType=@MonitorType, Address=@Address, GroupName=@Group, Name=@Name where Id=@Id", new { endpoint.MonitorType, endpoint.Address, endpoint.Group, endpoint.Name, endpoint.Id }, tx) == 0)

```

Comment:
if(conn.Execute($"update EndpointConfig set MonitorType=@MonitorType, Address=@Address, GroupName=@Group, Name=@Name{tags!=null?", Tags=@tags":""} where Id=@Id", new { endpoint.MonitorType, endpoint.Address, endpoint.Group, endpoint.Name, endpoint.Id, tags=entity.Tags }, tx) == 0)

then it would be enough to live the original 2 lines with insert extended with Tags